cfme/tests/automate/test_method.py

```diff
@@ -222,3 +225,51 @@ def test_automate_simulate_retry(klass, domain, namespace, original_class):
 
     # Checking whether 'Retry' button is displayed
     assert view.retry_button.is_displayed
+
+
+@pytest.fixture(scope='module')
+def service_catalog_item(request, dialog, catalog):
```

Comment:
I think you are searching for
https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/service_fixtures.py#L40
other type also available have a glance these service fixtures may be help you. Let me know.

Reply:
@digitronik 
- I went through two fixtures which return catalog item: ```catalog_item``` and ```generic_catalog_item```
- ```catalog_item``` requires provider and I am here creating ```generic``` catalog item. So I cant use this fixture.
- ```generic_catalog_item``` fixture yields the object which does not allow me to order catalog item via rest. This object required to be order via UI. So this fixture also is not useful here.
- Hence keeping as it is.